CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+# 0.9.1
+
+## Performance
+
+- **M3DB**: Use index aggregate query to speed up the [/label/<label_name>/values endpoint](https://prometheus.io/docs/prometheus/latest/querying/api/#querying-label-values) (#1628)
+
+## Bug Fixes
+
+- **M3Query**: Fix step parsing, previously if you included a float step size with no units the query would fail (#1617)
```

Comment:
`Fixed an issue causing untyped step sizes to fail rather than defaulting to seconds`